Identity/Identity.Business/Services/UserService.cs

```diff
@@ -52,143 +53,150 @@ public async Task<TokenDto> Login(LoginDto loginModel)
 	}
 
 
-	public async Task<UserDto> Register(RegisterDto registerDto, string? registererRole)
+	public async Task<UserDto> RegisterAsync(RegisterDto registerDto, string? receptionistRole)
 	{
-		if (registererRole is null)
+		var validator = new RegisterValidator();
+		await validator.ValidateAndThrowAsync(registerDto);
```

Comment:
you can use automatic validation and in services you wont have to create instanes of validators. read about AddFluentValidationAutoValidation method.